src/scala/ly/stealth/mesos/kafka/Scheduler.scala

```diff
@@ -59,7 +59,8 @@ object Scheduler extends org.apache.mesos.Scheduler {
     ExecutorInfo.newBuilder()
       .setExecutorId(ExecutorID.newBuilder.setValue(Broker.nextExecutorId(broker)))
       .setCommand(commandBuilder)
-      .setName("broker-" + broker.id)
+      .setName("kafka.broker")
+      .setSource(Config.frameworkName + ".broker." + broker.id)
```

Comment:
optional string source = 10 [deprecated = true]; // Since 1.0.

not sure what might be best here now